packages/block-library/src/quote/editor.scss

```diff
@@ -1,7 +1,47 @@
 .wp-block-quote {
-	margin: 0;
+	margin: 0 inherit;
```

Comment:
I had to be presumptuous here. I've seen too many inexperienced CSS authors be overly explicit in learning to write CSS with shorthand properties. 

Further reading: [Harry Roberts "CSS Shorthand Syntax Considered an Anti-Pattern"](https://csswizardry.com/2016/12/css-shorthand-syntax-considered-an-anti-pattern/)

The left/right margin seemed overly explicit, unnecessary, and would cause more headaches for theme authors to overwrite with `!important` further down the line. I agree that there is potential for "unanticipated regressions" now. However, I project that the possibility of these regressions now within the Gutenberg editor vs the outputted styling and front-end experience would outweigh the potential to allow this overly explicit styling to remain.

Reply:
@sarahmonster thanks for the great feedback. I truly appreciate it.

I tried to provide further reasoning behind my decisions. I would hope and encourage that we get another party's insight. I'm still new to the Gutenberg codebase and I don't want to miss anything in the larger scheme of things.

I hope that my reasoning is not perceived as a "no". I'm mostly just curious what more people think as I'm new to the project and tried to apply my most sound reasoning and experience, while also perusing existing Issues, Git Blame, and general history to get a sense of best practices and approaches. 

Hopefully, somebody else will drop some feedback and I can get all this updated in another pass, sooner than later. Thanks again!

Reply:
I would agree with Sarah here that this is a little sensitive to introduce at this point, which is right before the 5.0 release. Which means we have to be extra careful not to cause regressions. In this case it's important to test this both with themes that _do_ support wide alignments, and with themes that do not. There's also mobile, tablet, desktop breakpoints to consider, and even though these are editor styles, the frontend should ideally match. 

I understand your arguments and reasons for changing this — and I think this would make a good separate pull request that looks at these editor specific margin rules for _all the blocks, together_. But changing this for just the quote block in isolation is just going to cause inconsistency, even if this becomes the only block that's "right". 

In other words, unless this rule is _strictly necessary_ for the purpose of this PR, I would not bundle it into this PR.

Reply:
@jasmussen @sarahmonster

Thanks for the thorough and insightful review!

One thing still remains unclear. There are several mentions of “causing regressions”. Perhaps I’m missing some oversight on the Gutenberg project’s milestones and project goals. Since it has not been “officially” released yet then why are we concerned with regressions?

Reply:
> One thing still remains unclear. There are several mentions of “causing regressions”. Perhaps I’m missing some oversight on the Gutenberg project’s milestones and project goals. Since it has not been “officially” released yet then why are we concerned with regressions?

You mentioned yourself wanting to get these improvements in the 5.0 release. That release is due November 27th, which is two weeks from now. Which means any change we make between now and then has to be _solid_ and _stable_, and, well, not cause regressions. 

A regression could be:

- A theme that wants to be Gutenberg aware when 5.0 launches, and has perhaps optimized towards the current implementation of the Quote block, such as TwentyNineteen. (Though we'd probably be able to fix up _that_ specific theme beforehand, but the point is still valid for non twenty themes).
- Regarding the alignments, any existing post authored and saved before this change, when merged, might present an "invalid content" box, or show up as being authored in the Classic block, unless we create a deprecation handler. Creating a deprecation handler is possible, but it's not easy, and it's a relatively big thing to do which means it needs to be thoroughly reviewed and tested, which is hard given there are a lot of other open isssues in the [milestones](https://github.com/WordPress/gutenberg/milestones) we need to fix for the 5.0 release.
- If a left/right change is made, it could cause a regression for RTL languages where a different default might be wanted.
- If a change to margins is made, it could cause display issues on mobile, or in wide/full-wide configurations. For example, if you put a quote inside a full-wide columns block 

This is not to say any of those regressions can't be caught and addressed in this PR. It's just that if this PR has a goal of getting merged before the 5.0 release, then it will be held to a higher standard (and should probably be reduced in scope) than if, for example, it goes into the 5.1 release, at which point it's totally fair to expand the scope.

Reply:
@jasmussen thanks for the clarification. Makes absolute sense.